activerecord/test/fixtures/encrypted_book_that_ignores_cases.yml

```diff
@@ -1,5 +1,4 @@
 rfr:
   author_id: 1
-  id: 2
```

Comment:
Seeing some intermittent test failures like [this one](https://buildkite.com/rails/rails/builds/76506#f42a5ab3-1923-4a9f-b843-56c4098b6bef/1063-1075). Not sure about the cause. Hopefully, removing fixed ids will help.